### PR TITLE
Billing > Subaccounts activation

### PR DESCRIPTION
**Pull request by Ninjeneer**

### CODE REVIEW

File: src/types/Server.ts [hunk 1]
```diff
@@ -453,6 +453,7 @@ export enum ServerAction {
   BILLING_TEST_DATA_CLEANUP = 'BillingTestDataCleanup',
   BILLING_BILL_PENDING_TRANSACTION = 'BillingBillPendingTransaction',
   BILLING_SUB_ACCOUNT_CREATE = 'BillingCreateSubAccount',
+  BILLING_SUB_ACCOUNT_ACTIVATE = 'BillingSubAccountActivate',
```
Comment (ClaudeROSSI) [inline src/types/Server.ts:456]:
```suggestion
  BILLING_SUB_ACCOUNT_ACTIVATE = 'BillingActivateSubAccount',
```

File: src/assets/server/notification/email/de/billing-sub-account-activated.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "subject": "Billing - Connected account activated",
```
Comment (ClaudeROSSI) [inline src/assets/server/notification/email/de/billing-sub-account-activated.json:2]:
```suggestion
  "subject": "Billing account activated",
```

I would like to change the title. Is it ok with you if we simply put :
"Billing account activated" 

The user already knows that this is related to his own "connected" account

File: src/assets/server/notification/email/de/billing-sub-account-activated.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "tenant": "<%- tenantID %>",
+  "baseURL": "<%- evseDashboardURL %>",
+  "body": {
+    "header": {
+      "title": "Billing - Connected account activated",
```
Comment (ClaudeROSSI) [inline src/assets/server/notification/email/de/billing-sub-account-activated.json:7]: idem

File: src/assets/server/notification/email/fr/billing-sub-account-activated.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+  "subject": "Billing - Compte connecté activé",
```
Comment (ClaudeROSSI) [inline src/assets/server/notification/email/fr/billing-sub-account-activated.json:2]:
```suggestion
  "subject": "Compte de facturation activé",
```

File: src/assets/server/notification/email/fr/billing-sub-account-activated.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  "tenant": "<%- tenantID %>",
+  "baseURL": "<%- evseDashboardURL %>",
+  "body": {
+    "header": {
+      "title": "Billing - Compte connecté activé",
```
Comment (ClaudeROSSI) [inline src/assets/server/notification/email/fr/billing-sub-account-activated.json:7]: idem

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
@Ninjeneer : just some i18n comments regarding the mails and notificationds.
let me know what you think.

File: src/types/Server.ts (hunk 1 above)
Comment (Ninjeneer) [reply src/types/Server.ts:456]: I just wanted to keep the same naming convention, as we often use "EntityAction" format (TagCreate, UserImport, etc....)

File: src/assets/server/notification/email/de/billing-sub-account-activated.json (hunk 2 above)
Comment (Ninjeneer) [reply src/assets/server/notification/email/de/billing-sub-account-activated.json:2]: Indeed, you're right. Should I also remove the "Connected" mention in the Sub-account created notification ?

File: src/types/Server.ts (hunk 1 above)
Comment (ClaudeROSSI) [reply src/types/Server.ts:456]: > I just wanted to keep the same naming convention, as we often use "EntityAction" format (TagCreate, UserImport, etc....)

as you like ... just make sure to have the same logic for 'BillingCreateSubAccount' then.
Thanks

**Review by ClaudeROSSI — COMMENTED** [review]
@Ninjeneer : tests are not running on my side.
It sounds like the stripe account must be configured to accept sub-accounts ... 
let's have a call to discuss this

**Review by ClaudeROSSI — APPROVED** [review]

**Pull request merged**